README.Rmd

```diff
@@ -22,7 +22,7 @@ knitr::opts_chunk$set(
 [![CRAN status](https://www.r-pkg.org/badges/version/finalsize)](https://CRAN.R-project.org/package=finalsize)
 <!-- badges: end -->
 
-_finalsize_ provides calculations for the final size of an epidemic in a population with demographic variation in contact patterns, and variation within and between age groups in their susceptibility to disease.
+_finalsize_ provides calculations for the final size of an epidemic in a population, which is the overall number of infected individuals, depending on a demographic distribution (e.g., age groups) and their contact patterns, accounting for different susceptibility to disease between groups (e.g., due to age-group specific immune responses, such as age-related reduced immune function, or immunity from prior exposure to similar pathogens) and within groups (e.g., due to immunisation programs).
```

Comment:
To keep it short and explanatory enough, I would only remove "`, such as age-related reduced immune function, or immunity from prior exposure to similar pathogens`". Details in the primer section expand on this.

Reply:
Fixed in https://github.com/epiverse-trace/finalsize/pull/121/commits/eed6b9ced74e34ce334f1e17e536cb36f6c408cb